Nodejs/Setup/setup.proj

```diff
@@ -19,6 +19,7 @@
     <ProjectFile Include="NodejsTools\NodejsTools.wixproj"/>
     <ProjectFile Include="NodejsToolsInstaller\NodejsToolsInstaller.wixproj"/>
     <ProjectFile Include="..\..\Internal\Nodejs\VsLoggerSetup\VsLogger.wixproj" Condition="'$(IncludeVsLogger)' == 'True'"/>
+    <ProjectFile Include="Custom_Action\Custom_Action.csproj"/>
```

Comment:
Make this conditional. We only want to log installers that are actually being released, not random internal builds.